vendor/github.com/containerd/cri/vendor.conf

```diff
@@ -1,99 +1,99 @@
 # cri dependencies
 github.com/docker/docker                            4634ce647cf2ce2c6031129ccd109e557244986f
-github.com/opencontainers/selinux                   0d49ba2a6aae052c614dfe5de62a158711a6c461 # v1.5.1
-github.com/tchap/go-patricia                        666120de432aea38ab06bd5c818f04f4129882c9 # v2.2.6
+github.com/opencontainers/selinux                   v1.5.2
```

Comment:
Can you update the selinux dependency as well? (there was a fix for a race condition in the v1.5.2 update)

(not sure if you want to update the go-cni dependency to match as well https://github.com/containerd/containerd/pull/4266/files)

Reply:
Ok, let me tag the go-cni dep for gomod stuff